src/wp-includes/rest-api/endpoints/class-wp-rest-blocks-controller.php

```diff
@@ -40,6 +40,7 @@ public function check_read_permission( $post ) {
 	 * Filters a response based on the context defined in the schema.
 	 *
 	 * @since 5.0.0
+	 * @since 6.3 Adds the `wp_pattern_sync_status` postmeta property to the toplevel of response.
```

Comment:
```suggestion
	 * @since 6.3 Adds the `wp_pattern_sync_status` postmeta property to the top level of response.
```

Tiniest of nits: Missing space.

Reply:
Done.